widgets/service-request-form/src/App.vue

```diff
@@ -0,0 +1,357 @@
+            },
+            body: JSON.stringify({
+                code: formData.code,
+            }),
+        })
+            .then((response) => response.json())
+            .then((json) => {
+                if (json.errors) {
+                    node.setErrors([], json.errors);
+
+                    return;
+                }
+
+                if (json.is_expired) {
+                    node.setErrors(['The authentication code expires after 24 hours. Please authenticate again.']);
+
+                    authentication.value.isRequested = false;
+                    authentication.value.requestedMessage = null;
+
+                    return;
+                }
+
+                if (!json.submission_url) {
+                    node.setErrors([json.message]);
+
+                    return;
+                }
+
+                formSubmissionUrl.value = json.submission_url;
+            })
+            .catch((error) => {
+                node.setErrors([error]);
+            });
+
+        return;
+    }
+
+    fetch(authentication.value.requestUrl, {
+        method: 'POST',
+        headers: {
+            Accept: 'application/json',
+            'Content-Type': 'application/json',
+        },
+        body: JSON.stringify({
+            email: formData.email,
+        }),
+    })
+        .then((response) => response.json())
+        .then((json) => {
+            if (json.errors) {
+                node.setErrors([], json.errors);
+
+                return;
+            }
+
+            if (!json.authentication_url) {
+                node.setErrors([json.message]);
+
+                return;
+            }
+
+            authentication.value.isRequested = true;
+            authentication.value.requestedMessage = json.message;
+            authentication.value.url = json.authentication_url;
+        })
+        .catch((error) => {
+            node.setErrors([error]);
+        });
+}
+</script>
+
+<template>
+    <div
+        :style="{
+            '--primary-50': formPrimaryColor[50],
+            '--primary-100': formPrimaryColor[100],
+            '--primary-200': formPrimaryColor[200],
+            '--primary-300': formPrimaryColor[300],
+            '--primary-400': formPrimaryColor[400],
+            '--primary-500': formPrimaryColor[500],
+            '--primary-600': formPrimaryColor[600],
+            '--primary-700': formPrimaryColor[700],
+            '--primary-800': formPrimaryColor[800],
+            '--primary-900': formPrimaryColor[900],
+            '--rounding-sm': formRounding.sm,
+            '--rounding': formRounding.default,
+            '--rounding-md': formRounding.md,
+            '--rounding-lg': formRounding.lg,
+            '--rounding-full': formRounding.full,
+        }"
+        class="font-sans"
+    >
+        <div class="prose max-w-none" v-if="display && !submittedSuccess">
+            <link rel="stylesheet" v-bind:href="hostUrl + '/js/widgets/form/style.css'" />
```

Comment:
I believe this needs to be changed to point to it's own style sheet.

If you copied it from surveys, it might need to be adjusted as well.

I had to end up fixing this in events widget